pkg/tsdb/cloudwatch/credentials.go

```diff
@@ -177,11 +182,27 @@ func (e *CloudWatchExecutor) getAwsConfig(dsInfo *DatasourceInfo) (*aws.Config,
 		return nil, err
 	}
 
-	cfg := &aws.Config{
-		Region:      aws.String(dsInfo.Region),
-		Credentials: creds,
+	if dsInfo.TlsSkipVerify {
+		tlsCfg := &tls.Config{}
+		tlsCfg.InsecureSkipVerify = true
+
+		cfg := &aws.Config{
+			Region:      aws.String(dsInfo.Region),
+			Credentials: creds,
+			HTTPClient: &http.Client{
```

Comment:
think this should be changed to use the http client returned here: https://github.com/grafana/grafana/blob/1165d098b0d0ae705955f9d2ea104beea98ca6eb/pkg/models/datasource_cache.go#L28 

It has all the correct settings and transport and is what the other data sources use. Then you can remove this if case here.

Reply:
This is an excellent idea. Switching over to use this HttpClient makes the modifications much simpler and robust. I've gone ahead and updated my branch to use this suggestion.